distgen/README.md

```diff
@@ -43,3 +43,18 @@ global variables to generate. (See an example of a JSON file at
 package dist
 //go:generate go run github.com/chef/go-libs/distgen global.go dist https://example.com/path/to/glob_dist.json
 ```
+
+### Using an Environment Variable
+
+The usage of an environmental variable allows for better flexibility when dealing with CI systems.  
+The usage of an environmental variable allows for better flexibility when dealing with
+CI systems. As we saw in the previous example, the 3rd argument that is passed to
+`go run` is a URI, which is used to override the default JSON file for `distgen`. The
+`DIST_FILE` environment performs the same functionality by overloading the default
+JSON file. The usage inside of your dist files will not change, but you can pass ENV
```

Comment:
```suggestion
The usage inside of your dist files will not change, but you can pass ENV as a part of your `go generate` command.
```

Can you clarify  _what_ is used in the dist files?  Do you mean the URI? Something else?